From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Sat, 16 Nov 2024 13:21:27 +0100
Subject: Install wheel, not source

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 27f2d0b..e150cc7 100644
--- a/Makefile
+++ b/Makefile
@@ -42,8 +42,8 @@ dist:
 	python -m build .
 
 PIP ?= pip
-install:
+install-src:
 	$(PIP) install .
 
-install-dist:
+install:
 	$(PIP) install -I $(shell ls -1rt dist/*.whl | tail -n 1)
